skyVegasTests/gamePlayGameClient.spec.ts

```diff
@@ -1,35 +1,205 @@
-import { test, type Page } from '@playwright/test';
+import { test, type Page, expect } from '@playwright/test';
```

Comment:
The test is already quite complex and will become a debugging nightmare if we combine the two with all the calculations that have to be performed so that's why we opted to separate them out.

In terms of it being stuck on one test, it won't be , because it will reach a timeout.  And Playwright supports test retries. When enabled, failing tests will be retried multiple times until they pass, or until the maximum number of retries is reached.

For the win scenario we want to spin until we get a win as that's the point of the test. We can't just fail the test pack after 10 retries. So attempting to spin for 5 minutes and setting up test retries will statistically give us a very high chance to get a win and remove the flakiness of the test.